README.md

```diff
@@ -127,6 +166,18 @@ Here is an example of invoking the action with all required parameters:
 wsk action invoke /messaging/kafkaProduce -p brokers "[\"mykafkahost:9092\", \"mykafkahost:9093\"]" -p topic mytopic -p value "This is the content of my message"
 ```
 
+### Producing Messages to a Message Hub or Generic Kafka Instance using DIY action
```

Comment:
I'm not sure I understand the purpose of this section. It kind of seems unrelated to the messaging package in that it says "you really don't need this now, but just in case, here is an example of how you _used to_ do things."

It is a nice example though, so I wonder if maybe we just turn this into a more generic "Examples" section with a very brief (one sentence) description of the sample with a link to the GitHub project. If that becomes the case, then the previous two section titles can go back to the way they were before (without the "using system provided action").